build-tools/webpack/assets-writer-plugin.js

```diff
@@ -28,7 +28,7 @@ Object.assign( AssetsWriter.prototype, {
 	apply: function ( compiler ) {
 		const self = this;
 
-		compiler.hooks.afterEmit.tapAsync( 'AssetsWriter', ( compilation, callback ) => {
+		compiler.hooks.emit.tapAsync( 'AssetsWriter', ( compilation, callback ) => {
```

Comment:
https://webpack.js.org/blog/2020-10-10-webpack-5-release/#sizeonlysource-after-emit

A plugin running in `afterEmit` doesn't have access to `compilation.assets[ asset ].source()`